packages/containers/package.json

```diff
@@ -31,6 +31,18 @@
     "type": "git",
     "url": "https://github.com/Talend/ui.git"
   },
+  "dependencies": {
+    "bson-objectid": "^1.1.5",
+    "classnames": "^2.2.5",
+    "immutable": "^3.8.1",
+    "invariant": "^2.2.2",
+    "keycode": "^2.2.0",
+    "lodash": "^4.17.4",
+    "memoize-one": "^4.0.0",
```

Comment:
Little tought, maybe we should use memo or reselect for this need ( not to change in this pr )

Reply:
In this case, it's more reselect. It's in a class, so useMemo is not an option, and in the class the usage is to convert form schemas from immutable to js.
But as you meant, it's out of the scope